.rubocop.yml

```diff
@@ -0,0 +1,37 @@
+require: rubocop-rspec
+
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - 'vendor/**/*'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'app/assets/**/*'
+    - 'config/**/*'
+    - 'doc/**/*'
+    - 'spec/**/*'
+    - 'test/**/*'
+
+Rails:
+    Enabled: true
+
+Style/Encoding:
+  Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
+
+Style/Documentation:
+  Exclude:
+    - 'spec/**/*'
+    - 'app/**/*'
+
+Style/Next:
+  Enabled: false
```

Comment:
@mcritchlow Sounds fine to me, I will enable it now as you all voted it.
Just provide some of other folk's experience so that we could realize both sides of it.
And as you mentioned, we could always disable it if it is getting annoying.